po/de.po

```diff
@@ -22736,11 +22736,11 @@ msgstr "Entpacken der Tree-Objekte fehlgeschlagen"
 
 #: builtin/stash.c:839
 msgid "include untracked files in the stash"
-msgstr "unversionierte Dateien in Stash einbeziehen"
+msgstr "unversionierte Dateien im Stash einbeziehen"
```

Comment:
Ich denke das war schon richtig so. Wg. "in (den) Stash einbeziehen". So wie "Ich zeige etwas im Stash" und "Ich schreibe etwas in (den) Stash.".

Reply:
ok 😯